lib/src/hd_wallet.dart

```diff
@@ -31,7 +31,11 @@ class StellarHDWallet {
       {int strength = ENTROPY_BITS_HALF,
       language = 'english',
       bip39.RandomBytes random}) {
-    return bip39.generateMnemonic(strength: strength, randomBytes: random);
+    if (random is Function) {
```

Comment:
Technically, this could be:
```
if (random is bip39.RandomBytes) {
```
but there is no functional difference in being so specific, as the typing will already pick this up.
It could also be made simpler as:
```
if (random != null) {
```
But a null check looks ugly.